minicluster/pom.xml

```diff
@@ -19,8 +19,30 @@
     <findbugs.path>${project.parent.basedir}/build/findbugs/</findbugs.path>
   </properties>
 
+  <build>
+    <plugins>
+     <plugin>
+       <groupId>org.apache.maven.plugins</groupId>
+       <artifactId>maven-jar-plugin</artifactId>
+       <version>2.2</version>
```

Comment:
I think we don 't need line 27 as in parent pom file, `maven-jar-plugin` has been specified in `<pluginManagement>`  to use version 2.4